posthog/views.py

```diff
@@ -76,9 +72,20 @@ def stats(request):
 
 
 def robots_txt(request):
+    ROBOTS_TXT_CONTENT = (
+        "User-agent: *\nDisallow: /shared_dashboard/" if settings.MULTI_TENANCY else "User-agent: *\nDisallow: /"
+    )
     return HttpResponse(ROBOTS_TXT_CONTENT, content_type="text/plain")
 
 
+def security_txt(request):
+    SECURITY_TXT_CONTENT = """
+        Contact: engineering@posthog.com
+        Hiring: https://posthog.com/careers
```

Comment:
2 year expire buffer? or should we revisit this once a year? @Twixes 

```suggestion
        Contact: mailto:engineering@posthog.com
        Hiring: https://posthog.com/careers
        Expires: 2024-03-14T00:00:00.000Z
```